reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
```

Comment:
Yes we can do that. By default, most Gerrit APIs return JSON response. But to be sure, I will add Content-Type in header.